### DISCUSSION TITLE
Assign ID on group of radios

### DISCUSSION THREAD

**Discussion by stijn1989** [phalcon/cphalcon#15592] [Q&A]
https://github.com/phalcon/cphalcon/blob/44aa04c9871b4d9f14c8815b3b46df1817316074/phalcon/Tag.zep#L1473

I have a group of radios, which has the same name but different value. But they get the same ID. This gives a warning in the browser console.

One question: why?

**Comment by Jeckerson** [comment]
Hello

You can specify manually the id during radio input construction.

**Reply by stijn1989** [reply]
I did this but I shouldn't do it. If I don't specifiy an ID attribute for an element, then I expect an element without ID. Now the framework is adding this. Also you can't find this in the docs.

**Reply by wurst-hans** [reply]
Often I use a group of radio buttons with a dynamic set of values. For this, I misuse the `Select` element along with a custom Volt partial which assigns random IDs to each radio button and its label.
